.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,5 +1,8 @@
-name: Problems with a script
-description: Generate a report on an issue pertaining specifically to a script. For other inquires please share them in the Discussions section.
+name: "🐞 Script Issue Report"
+description: Report a specific issue with a script. For general inquiries, please use the Discussions section.
+title: "[Issue]: "
+labels: "Status: Unconfirmed"
```

Comment:
```suggestion
```
We shouldn't overlabel things. No labels is a good indication no one has looked at the issue yet.